Dockerfile.starter

```diff
@@ -1,5 +1,3 @@
-FROM radial/busyboxplus:curl
+FROM curlimages/curl:latest
 
-# as defined by distroless for nonroot
-USER 65532:65532
-COPY --from=redboxoss/scuttle:latest /scuttle /bin/scuttle
+COPY --from=kvij/scuttle:latest /scuttle /bin/scuttle
```

Comment:
`nonroot` user definitely shouldn't be removed here. 
On that note, are these two Docker changes necessary for ARM support? If they aren't, again, I'd suggest moving them out into a separate PR.

Reply:
yes, those are required for ARM64 support, as the current ones are old/unmaintained and lacks arm64 support. is the USER really necessary for curlimages/curl:latest(https://github.com/curl/curl-container)?

Reply:
>  those are required for ARM64 support, as the current ones are old/unmaintained and lacks arm64 support
OK, thanks for clarification.

> is the USER really necessary for curlimages/curl:latest(https://github.com/curl/curl-container)?
Yes. It was added deliberately. Currently it is present in both controller and starter; please keep those.

Reply:
>  Currently it is present in both controller and starter; please keep those.

added USER 65532:65532 now